recipes/jungle/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, patch
+from conans import CMake
+from conans.tools import check_min_cppstd
+
+required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
 from conans.tools import check_min_cppstd requires 1.50